### PR TITLE
Feat/app react

### LINKED ISSUE #21: UI Front End

### PR DESCRIPTION
**Pull request by trevorwhealy**
closes #21

### CODE REVIEW

File: client/App.jsx [hunk 1]
```diff
@@ -2,40 +2,104 @@ import React from 'react';
...
+            <div className="col-xs-10">
+              {/* Create the urgentTask bubbles */}
+              {this.state.urgentTasks.map((urgentTask, i) => {
+                let urgentTaskName = urgentTask.name;
+                let urgentTaskDueBy = urgentTask.dueBy.toString();
+                console.log(urgentTaskDueBy);
+                return (
+                  <div className="col-xs-3">
+                    <Circle name={urgentTaskName} dueBy={urgentTaskDueBy} key={i}/>
+                  </div>
```
Comment (donovanCarreno) [inline client/App.jsx:67]: should this be abstracted out into it's own component (e.g. UrgentTaskView) that then has Circle children components?  Similar to how we had a VideoListView and VideoListEntry in Recast.ly?

File: client/App.jsx [hunk 2]
```diff
@@ -2,40 +2,104 @@ import React from 'react';
...
+          </div>
+
+          {/* Create the completedTasks cards */}
+          {this.state.completedTasks.map((completedTask, i) => {
+            let completedTaskName = completedTask.userId;
+            let completedTaskUser = completedTask.taskId;
+            return (
+              <Comp task={completedTaskName} user={completedTaskUser} key={i}/>
+            );
+          })}
```
Comment (donovanCarreno) [inline client/App.jsx:84]: Should this also be abstracted out into a separate component?

File: client/App.jsx (hunk 1 above)
Comment (trevorwhealy) [inline client/App.jsx:67]: Normally yes but React doesnt allow me to pass objects as children, so I have to expose the properties on the object first.

**Comment by donovanCarreno** [comment]
🚢

**Pull request merged**